<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUp</class>
 <widget class="QDialog" name="SignUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Reset some default styling */
* {
    margin: 0;
    padding: 0;
}

/* Body styles */
body {
    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
    background-color: #f8f9fa;
    color: #1e1e1e;
}

/* Container styles */
.container {
    max-width: 400px;
    margin: 50px auto;
    padding: 20px;
    background-color: #ffffff;
    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
    border-radius: 8px;
}

/* Label styles */
label {
    display: block;
    margin-bottom: 5px;
    font-size: 14px;
    color: #7d7d7d;
}

/* Input styles */
input[type=&quot;text&quot;],
input[type=&quot;password&quot;] {
    width: 100%;
    padding: 10px;
    margin-bottom: 15px;
    border: 1px solid #d3d3d3;
    border-radius: 4px;
    color: #1e1e1e;
    transition: border-color 0.3s ease;
}

input[type=&quot;text&quot;]:focus,
input[type=&quot;password&quot;]:focus {
    border-color: #4a90e2;
}

/* Button styles */
button {
    display: inline-block;
    padding: 12px 20px;
    font-size: 16px;
    font-weight: bold;
    text-align: center;
    text-decoration: none;
    background-color: #4a90e2;
    color: #ffffff;
    border: none;
    border-radius: 4px;
    cursor: pointer;
    transition: background-color 0.3s ease;
}

button:hover {
    background-color: #3465a4;
}

/* Additional styles */
.login-button {
    width: 100%;
}
</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="pushButton_SignUp">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>360</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Sign Up</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Title">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Identify yourself</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_SignIn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>440</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Sign In</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_HaveAccount">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>400</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Already have an account?</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>381</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_Name">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>14</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Name">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_Email">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>14</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Email">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_Password">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>14</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Password">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_Confirm">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Confirm</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Confirm">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
